tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py

```diff
@@ -71,8 +69,17 @@ def convert_coll_tag(coll_path):
     stubbs_tagdata  = stubbs_coll_tag.data.tagdata
 
     # move blocks from stubbs tag into the halo one
-    for i in (0, 1, 2, 3, 6, 7, 8):
-        tagdata[i] = stubbs_tagdata[i]
+    for attr_name in (
+            "flags",
+            "indirect_damage_material",
+            "body",
+            "shield",
+            "modifiers",
+            "pathfinding_box",
+            "pathfinding_spheres",
+            "nodes",
+            ):
+        tagdata[attr_name] = stubbs_tagdata[attr_name]
```

Comment:
If you can type these out you might as well directly assign them.

Reply:
i did test the stubbs converters before i sent them to you. i had to use them to help that one user in Reclaimers. That's how I noticed they crashed on compile.